Test/TestCase/SearchManagerTest.php

```diff
@@ -158,12 +159,49 @@ public function testRemove()
         $this->assertNull($view);
     }
 
+    public function testRemoveAll()
```

Comment:
Also need a test case for `remove()` called with just a single or array of entities.